moniker_src/data_moniker.proto

```diff
@@ -1,30 +1,17 @@
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
 syntax = "proto3";
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-import "google/protobuf/any.proto";
+option cc_enable_arenas = true;
+option csharp_namespace = "NationalInstruments.DataMonikers";
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-package ni.data_monikers.v1;
+package ni.data_monikers;
```

Comment:
We should follow our conventions and use .v1 in the package name.

Reply:
reverted!